web/src/app/selection/DestinationField.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { DestinationType, FieldValueInput } from '../../schema'
+import DestinationInputDirect from './DestinationInputDirect'
+import { useDestinationType } from '../util/RequireConfig'
+
+export type DestinationFieldProps = {
+  value: FieldValueInput[]
+  onChange?: (value: FieldValueInput[]) => void
+  destType: DestinationType
+
+  disabled?: boolean
+}
+
+export default function DestinationField(
+  props: DestinationFieldProps,
+): React.ReactNode {
+  const dest = useDestinationType(props.destType)
+
+  return dest.requiredFields.map((field) => {
+    const fieldValue =
+      (props.value || []).find((v) => v.fieldID === field.fieldID)?.value || ''
+
+    function handleChange(newValue: string): void {
+      if (!props.onChange) return
+
+      const newValues = (props.value || [])
+        .filter((v) => v.fieldID !== field.fieldID)
+        .concat({
+          fieldID: field.fieldID,
+          value: newValue,
+        })
+
+      props.onChange(newValues)
+    }
+
+    if (field.isSearchSelectable)
+      throw new Error('search select is not enabled')
```

Comment:
Does this bubble up and render in the UI? If so can we make this a bit more ux friendly?

```suggestion
      throw new Error('Search select is not enabled. Please contact your administrator.')
```

Reply:
This was just temporary until SearchSelect was merged